checklists/alz_checklist.en.json

```diff
@@ -1006,6 +1006,78 @@
             "severity": "High",
             "link": "https://learn.microsoft.com/azure/expressroute/designing-for-high-availability-with-expressroute#active-active-connections"
         },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Hybrid",
+            "text": "Ensure the two physical links of your ExpressRoute circuit are connected to two distinct edge devices in your network.",
+            "waf": "Reliability",
+            "service": "ExpressRoute",
+            "guid": "b258f058-b9f6-46cd-b28d-990106f0c3f8",
+            "id": "D04.19",
+            "severity": "Medium",
+            "training": "https://learn.microsoft.com/learn/modules/design-implement-azure-expressroute/",
+            "link": "https://learn.microsoft.com/en-us/azure/expressroute/designing-for-high-availability-with-expressroute"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Hybrid",
+            "text": "Ensure Bidirectional Forwarding Detection (BFD) is enabled and configured on customer or provider edge routing devices.",
+            "waf": "Reliability",
+            "service": "ExpressRoute",
+            "guid": "fe2a1b53-6fbd-4c67-b58a-85d7c7a0afcb",
+            "id": "D04.20",
+            "severity": "Medium",
+            "training": "https://learn.microsoft.com/learn/modules/design-implement-azure-expressroute/",
+            "link": "https://learn.microsoft.com/en-us/azure/expressroute/expressroute-bfd"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Operations",
+            "text": "Configure Service Health to receive ExpressRoute circuit maintenance notifications.",
+            "waf": "Operations",
+            "service": "ExpressRoute",
+            "guid": "0031374c-16ea-4d5c-ad7a-0ba6190dd2a2",
+            "id": "D04.21",
+            "severity": "Medium",
+            "training": "https://learn.microsoft.com/learn/modules/design-implement-azure-expressroute/",
+            "link": "https://learn.microsoft.com/en-us/azure/expressroute/maintenance-alerts"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Hybrid",
+            "text": "Connect the ExpressRoute Gateway to two or more circuits from different peering locations for higher resiliency.",
```

Comment:
Are we able to add something around leveraging zone redundant gateways in supported regions?

Reply:
Might be another check? Open to thoughts.

Reply:
@igorjnzl there is already an item for this, please give a look to D4.06. Thank you.